demos/fleet_provisioning/fleet_provisioning_csr/mqtt_operations.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * AWS IoT Device SDK for Embedded C 202103.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef MQTT_OPERATIONS_H_
+#define MQTT_OPERATIONS_H_
+
+/* MQTT API header. */
```

Comment:
Instead of the MQTT helper functions being copied across multiple demo folders, it would be better to keep them in a single place (under `demos/mqtt_helpers/` for example) to avoid maintenance overhead.

Reply:
Ones for this demo had to be modified to take parameters for cert/key

Reply:
That would only be for the connection operation, right? Should other operations of subscribe, publish, unsubscribe, etc, be affected? 
If there is only a single function that had to be modified, it would be better to add that to the single set of utilities for MQTT operations that are used by all demos...